web/src/scripts/webflow/webflowSync.test.js

```diff
@@ -23,6 +23,7 @@ import {
   getUpdatedSectorNames,
   updateSectorNames,
 } from "./sectorSync.mjs";
+import { fundingCollectionId, sectorCollectionId } from "./webflowIds";
 
 const adjustDateByDay = (dayValue) => {
   const d = new Date();
```

Comment:
There's a `TODO` on line 34 to create a generator. By golly does this need one.

I thought about doing it here, but because I wasn't confident what all the tests were trying to achieve I didn't want to mess with too many things at once. Just wanted to get in and get out.

Adding a generator should be required for any TS rewrite we do here.

Reply:
Another thing... I cannot tell you how much time I would have saved if only I had types.